{"name": "test_kuquguanli[case3]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = judge_expect({'data': 956, 'errCode': None, 'errMessage': None, 'success': True}, 'success=false', <requests.sessions.Session object at 0x000001EC495F7C70>, <common.do_excel.Case object at 0x000001EC495DF0D0>, <Response [200]>)\n +    where 'success=false' = <common.do_excel.Case object at 0x000001EC495DF0D0>.case_expected", "trace": "self = <testcases.test_wms.Test_wms_test object at 0x000001EC495DBAF0>\nlogin_system = <requests.sessions.Session object at 0x000001EC495F7C70>\ncase = <common.do_excel.Case object at 0x000001EC495DF0D0>\n\n    @pytest.mark.parametrize(\"case\", wms_case[\"库区管理\"])\n    def test_kuquguanli(self, login_system, case):\n        \"\"\"\n        执行库区管理的测试\n        :param login_system:\n        :param case:\n        :return:\n        \"\"\"\n        logger.info(get_start_sep(f\"正在执行库区管理的测试，用例id为：{case.case_id} 用例标题为：{case.case_title}  \"))\n        if case.case_method.lower() == \"get\":\n            res = login_system.get(case.case_url)\n            logger.info(get_setup_sep(\"正在发送 GET 请求\"))\n            res_json = res.json()\n            assert judge_expect(res_json, case.case_expected, login_system, case, res)\n        elif case.case_method.lower() == \"post\":\n            logger.info(get_setup_sep(\"正在发送POST请求\"))\n            data = json.loads(case.case_data)\n            res = login_system.post(case.case_url, json=data)\n            res_json = res.json()\n>           assert judge_expect(res_json, case.case_expected, login_system, case, res)\nE           AssertionError: assert False\nE            +  where False = judge_expect({'data': 956, 'errCode': None, 'errMessage': None, 'success': True}, 'success=false', <requests.sessions.Session object at 0x000001EC495F7C70>, <common.do_excel.Case object at 0x000001EC495DF0D0>, <Response [200]>)\nE            +    where 'success=false' = <common.do_excel.Case object at 0x000001EC495DF0D0>.case_expected\n\nE:\\python api\\auto_interface_test_1.0\\testcases\\test_wms.py:34: AssertionError"}, "description": "\n        执行库区管理的测试\n        :param login_system:\n        :param case:\n        :return:\n        ", "attachments": [{"name": "log", "source": "738c5f90-8916-4a14-a5ce-d3a1a29e8db5-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "<common.do_excel.Case object at 0x000001EC495DF0D0>"}], "start": 1664813822277, "stop": 1664813822351, "uuid": "f101678c-2064-4f3e-aa15-b7ee1b8c4193", "historyId": "82a522b2300927a70865e64f67545e08", "testCaseId": "569dfe3ccef05efc2369ab78a96ecaad", "fullName": "testcases.test_wms.Test_wms_test#test_kuquguanli", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_wms"}, {"name": "subSuite", "value": "Test_wms_test"}, {"name": "host", "value": "DESKTOP-3HSC7J4"}, {"name": "thread", "value": "12552-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_wms"}]}